blogposts/2021/dev-tool-time-roger-peppe.md

```diff
@@ -0,0 +1,72 @@
+
+With Acme as his primary dev environment, though, Rog needs a three button mouse, and while he's tried the [Logitech MX Master mouse](https://www.logitech.com/en-us/products/mice/mx-master-3.html), he prefers older, sturdier versions from Logitech.
+
+## Between familiarity and strangeness
+
+The quality of mouse, and it's reliability through heavy use, is important for Rog. Acme's interaction model is focused on mouse movements, clicks, and chords (combination clicks). While Vim, Emacs, and many other editors have focused on typing, key presses, and key chords for selections, movement around the editor, and actions, Acme presents a different model of engagement, one that is, in many ways familiar. Need to move your cursor to text in another Acme window? Just click there. Want to select text? Just drag while holding a mouse button, the way you do in so many other applications. Acme coheres with common mouse interactions. Yet, Acme is also a bit strange. Clicking with the right button doesn't open a contextual menu; it performs a contextual search that can be used for actions like opening files, finding the next instance of a word, or running a regex search. Clicking with the center button, the click-wheel on Rog's mouse, executes selected text.
+
+TODO: VIDEO EMBED
+
+As Rog shows, these patterns are remarkably powerful in Acme. In every window, there are sets of commands in what we might recognize as a menu- or toolbars. Yet, in Acme, you can write text anywhere and that text is executable. When Rog needs a new command, he writes the script, in Go, the rc shell, or another language, and can execute it easily just by typing the command name and clicking. Imagine working in Google Docs, opening an edit menu, and clicking on the custom command you wrote. This is strangeness and familiarity of Acme, which, originally part of the [Plan9 operating system](http://9p.io/plan9/), indicated a different way forward from our contemporary mainstream computing practices.
+
+## The simplicity of Acme
+
+For Rog, the driving benefit of Acme is its simplicity. As he notes, in Acme, everything is text and text is powerful. Any text could be executable, and you can type text almost anywhere in Acme. You aren't constrained to toolbars or a single command line, or even the mental model of there being a narrow area of UI that's interactive and command focused.
+
+TODO: EMBED A VIDEO OR BLOCKQUOTE OF ROG ON TEXT IS POWERFUL OR CLIP OF ROG "TALKING TO EDITOR"
+
+You also aren't bound to memorize a plethora of keybindings to make you more efficient. While many people talk about keybindings being efficient, Rog finds his mouse just as efficient, and, perhaps, simpler to use. Three buttons are far fewer than the keys on a keyboard, and the combinations (chords) more limited.
```

Comment:
```suggestion
You also don't have to memorize a plethora of keybindings to make you more efficient. While many people talk about keybindings being efficient, Rog finds his mouse is just as efficient and simpler to use. Three buttons are far fewer than the keys on a keyboard and the combinations (chords) are more limited.
```
I don't think we need to say "perhaps" if it being simpler is his opinion but let me know if I'm missing context from the recording.